### PR TITLE
[en] add a device plugin example 

### PR DESCRIPTION
**Pull request by liuyuanzhi-cn**

### CODE REVIEW

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -447,6 +447,7 @@ Here are some examples of device plugin implementations:
 * The [Solarflare device plugin](https://github.com/vikaschoudhary16/sfc-device-plugin)
 * The [SR-IOV Network device plugin](https://github.com/intel/sriov-network-device-plugin)
 * The [Xilinx FPGA device plugins](https://github.com/Xilinx/FPGA_as_a_Service/tree/master/k8s-device-plugin) for Xilinx FPGA devices
+* The [generic device plugin](https://github.com/squat/generic-device-plugin) for generic linux devices
```
Comment (Ritikaa96) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:450]:
```suggestion
* The [Generic device plugin](https://github.com/squat/generic-device-plugin) for generic Linux devices
* The [Intel device plugins](https://github.com/intel/intel-device-plugins-for-kubernetes) for
  Intel GPU, FPGA, QAT, VPU, SGX, DSA, DLB and IAA devices
* The [KubeVirt device plugins](https://github.com/kubevirt/kubernetes-device-plugins) for
  hardware-assisted virtualization
* The [NVIDIA GPU device plugin for Container-Optimized OS](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
* The [RDMA device plugin](https://github.com/hustcat/k8s-rdma-device-plugin)
* The [SocketCAN device plugin](https://github.com/collabora/k8s-socketcan)
* The [Solarflare device plugin](https://github.com/vikaschoudhary16/sfc-device-plugin)
* The [SR-IOV Network device plugin](https://github.com/intel/sriov-network-device-plugin)
* The [Xilinx FPGA device plugin](https://github.com/Xilinx/FPGA_as_a_Service/tree/master/k8s-device-plugin) for Xilinx FPGA devices
```

Comment (Ritikaa96) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:450]: small nits and Let's keep it Alphabetically right.

**Comment by utkarsh-singh1** [comment]
Hi @liuyuanzhi-cn, Thanks for raising the PR, please try to follow the template of PR as it will help reviewers to assess your contribution.

**Comment by utkarsh-singh1** [comment]
Also seems like there is netlify build timeout ,

please follow this guide,

```bash
git reset
git commit --amend --no-edit
git push --force-with-lease
```

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 2]
```diff
@@ -447,6 +447,17 @@ Here are some examples of device plugin implementations:
...
+* The [Intel device plugins](https://github.com/intel/intel-device-plugins-for-kubernetes) for
+  Intel GPU, FPGA, QAT, VPU, SGX, DSA, DLB and IAA devices
+* The [KubeVirt device plugins](https://github.com/kubevirt/kubernetes-device-plugins) for
+  hardware-assisted virtualization
+* The [NVIDIA GPU device plugin for Container-Optimized OS](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
+* The [RDMA device plugin](https://github.com/hustcat/k8s-rdma-device-plugin)
+* The [SocketCAN device plugin](https://github.com/collabora/k8s-socketcan)
+* The [Solarflare device plugin](https://github.com/vikaschoudhary16/sfc-device-plugin)
+* The [SR-IOV Network device plugin](https://github.com/intel/sriov-network-device-plugin)
+* The [Xilinx FPGA device plugin](https://github.com/Xilinx/FPGA_as_a_Service/tree/master/k8s-device-plugin) for Xilinx FPGA devices
```
Comment (tengqm) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:460]: Please merge the list.

**Review by windsonsea — COMMENTED** [review]
ping @liuyuanzhi-cn. This list has many duplicated items. Could you please remove them?

**Comment by tengqm** [comment]
No response for almost two months.
Closing this.
/close
Feel free to reopen and resume the work.

**Pull request closed without merging**